docs/build/js-client/3-hello-world.md

```diff
@@ -48,7 +48,7 @@ npm init
 We will need these three packages for the application runtime: JS Client API, JS Client implementation for Node.js, and the package containing a well-maintained list of relay nodes.
 
 ```sh
-npm install @fluencelabs/js-client.api @fluencelabs/js-client.node @fluencelabs/fluence-network-environment
+npm install @fluencelabs/js-client
```

Comment:
you need to be a lot more explicit with respect to both project goal, e.g., why are we doing what we're doing and how is that different from the rust peer approach

Reply:
I agree with the point of your message, but the line above this comment doesn't seem to relate to what you said. Probably you meant other places in docs?

Reply:
yeah. higher up. wouldn't let me tag the line and i failed to link the line number. sorry.

Reply:
"higher up" where exactly?